user-guide/Advanced_Functionality/Security/DataMiner_hardening_guide.md

```diff
@@ -140,23 +140,50 @@ For more information about disabling legacy SSL/TLS versions, refer to [TLS, DTL
 
 ### Configure the firewall
 
-On DataMiner versions installed using the **10.0 installer** (or older), the DataMiner installation opens the following (inbound) ports and rules in the Windows firewall:
+Depending on which version of the DataMiner Installer is used, different firewall ports are opened by default. You can find more information about this below.
 
-- TCP **23**: Telnet (disabled by default)
+#### [Installer v10.4](#tab/tabid-1)
+
+If DataMiner is installed with the **DataMiner Installer v10.4**, the following (inbound) ports and rules are opened in the Windows firewall:
 
 - TCP 80: HTTP
 
 - TCP 8004: Remoting (client-server & inter-DMS communication)
 
-- TCP **9004**: Web Services (end of life)
+- TCP 5100: CloudGateway (dataminer.services endpoint)
+
+- TCP 4222 and 6222: NATS (inter-process communication)
+
+- TCP 9090: NATS Account Server
+
+- UDP 162: SNMP
+
+- Allow ICMP: Ping
+
+Some of the ports that are opened by default can potentially be closed depending on the configuration of your DataMiner System:
+
+- TCP port **80** can be closed if IIS is configured to require HTTPS connections and if IIS is not configured to redirect HTTP to HTTPS. We highly recommended enabling HTTPS on your DataMiner System. Note that TCP port 443 needs to be open for HTTPS connections. For more information, see [Setting up https on a DMA](xref:Setting_up_HTTPS_on_a_DMA).
+
+- TCP port **8004** can be closed if the DMA is configured to use gRPC for both Cube-to-DMA and inter-DMA communication.
+
+- The ports for NATS communication, i.e. **4222, 6222, and 9090**, can be closed when the DMA is not part of a cluster.
+
+- TCP port **5100** can be closed when the DMA is not part of a cluster and no DxMs are hosted on external machines.
+
+#### [Installer v10.1](#tab/tabid-2)
+
+On DataMiner versions installed using the **DataMiner Installer v10.1**, the DataMiner installation opens the following (inbound) ports and rules in the Windows firewall:
+
+- TCP 80: HTTP
+
+- TCP 8004: Remoting (client-server & inter-DMS communication)
 
 - TCP 7000: Cassandra (inter-node communication)
 
 - TCP 9042: Cassandra (client-server communication)
 
   > [!NOTE]
-  > This rule and the one above for TCP 7000 only apply when the DataMiner System uses a Cassandra database locally.
-  > If Cassandra is configured to use TLS, port 7001 and 9142 are used instead. For detailed information on securing Cassandra, refer to [securing self hosted storage](#secure-self-hosted-dataminer-storage).
+  > This rule and the one above for TCP 7000 only apply when the DataMiner System uses a Cassandra database locally. If Cassandra is configured to use TLS, port 7001 and 9142 are used instead. For detailed information on securing Cassandra, refer to [securing self hosted storage](#secure-self-hosted-dataminer-storage).
```

Comment:
What is meant with "locally" (also below)? We may need to clarify this, because this could mean that the Cassandra DB is present on the same server as DataMiner, or that a Cassandra DB per DMA is used, or something else still.